### PR TITLE
Ingest csv review - HYPM

### PR DESCRIPTION
**Pull request by cdobs**
Completed review of HYPM telemetered and recovered ingestion csvs

### CODE REVIEW

File: GI02HYPM/GI02HYPM_R00005_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00005/instruments/CTDMO_sn*/*.hex,GI02HYPM-RIM01-02-CTDMOG039,recovered_inst,Available,
+#mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00005/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-00-WFPENG000,recovered_wfp,Pending,unable to parse. Redmine ticket 12792
+mi.dataset.driver.flord_l_wfp.flord_l_wfp_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00005/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-01-FLORDL000,recovered_wfp,Available,
+mi.dataset.driver.dosta_ln.wfp.dosta_ln_wfp_driver,/omc_data/whoi/OMC/GI02HYPM/R00005/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-03-DOSTAL000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00005/instruments/GWFP_sn*/0*/C*.DA?,GI02HYPM-WFP02-04-CTDPFL000,recovered_wfp,Available,
+mi.dataset.driver.vel3d_l.wfp.vel3d_l_wfp_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00005/instruments/GWFP_sn*/0*/A*.DA?,GI02HYPM-WFP02-05-VEL3DL000,recovered_wfp,Available,
```
Comment (snwhite) [inline GI02HYPM/GI02HYPM_R00005_ingest.csv:11]: Looks good.

File: GI02HYPM/GI02HYPM_R00006_ingest.csv [hunk 2]
```diff
@@ -4,8 +4,8 @@ parser,filename_mask,reference_designator,data_source,status,notes
 mi.dataset.driver.sio_eng.sio.sio_eng_sio_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00006/cg_data/MSIOC_sn*/STATUS/STA*.DA?,GI02HYPM-RIM01-00-SIOENG000,recovered_host,Available,
 mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_sio_ct_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00006/cg_data/MSIOC_sn*/CTD/*.DAT,GI02HYPM-RIM01-02-CTDMOG039,recovered_host,Available,
 mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00006/instruments/CTDMO_sn*/*.hex,GI02HYPM-RIM01-02-CTDMOG039,recovered_inst,Available,
-mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00006/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-00-WFPENG000,recovered_wfp,Available,
+#mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GI02HYPM/R00006/instruments/GWFP_sn*/0*/E*.DA?,GI02HYPM-WFP02-00-WFPENG000,recovered_wfp,Pending,unable to parse. Redmine ticket 12792
```
Comment (snwhite) [inline GI02HYPM/GI02HYPM_R00006_ingest.csv:7]: Looks good.

File: GP02HYPM/GP02HYPM_D00007_ingest.csv [hunk 3]
```diff
@@ -3,13 +3,13 @@ parser,filename_mask,reference_designator,data_source,status,notes
...
+#mi.dataset.driver.WFP_ENG.wfp_sio.wfp_eng_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00007/*.we_wfp_1398821.dat,GP02HYPM-WFP02-00-WFPENG000,telemetered,Not Expected,No mule data from upper profiler
+#mi.dataset.driver.flord_l_wfp.sio.flord_l_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00007/*.we_wfp_1398821.dat,GP02HYPM-WFP02-01-FLORDL000,telemetered,Available,No mule data from upper profiler
+#mi.dataset.driver.dosta_ln.wfp_sio.dosta_ln_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00007/*.we_wfp_1398821.dat,GP02HYPM-WFP02-03-DOSTAL000,telemetered,Available,No mule data from upper profiler
+#mi.dataset.driver.ctdpf_ckl.wfp_sio.ctdpf_ckl_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00007/*.wc_wfp_1398821.dat,GP02HYPM-WFP02-04-CTDPFL000,telemetered,Available,No mule data from upper profiler
+#mi.dataset.driver.vel3d_l.wfp.sio.vel3d_l_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GP02HYPM/D00007/*.wa_wfp_1398821.dat,GP02HYPM-WFP02-05-VEL3DL000,telemetered,Available,No mule data from upper profiler
```
Comment (snwhite) [inline GP02HYPM/GP02HYPM_D00007_ingest.csv:10]: Looks good.

File: GP02HYPM/GP02HYPM_R00005_ingest.csv [hunk 4]
```diff
@@ -8,9 +8,9 @@ mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/O
 mi.dataset.driver.flord_l_wfp.flord_l_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00005/instruments/GWFP_sn12997-01/0*/E*.DA?,GP02HYPM-WFP02-01-FLORDL000,recovered_wfp,Available,
 mi.dataset.driver.dosta_ln.wfp.dosta_ln_wfp_driver,/omc_data/whoi/OMC/GP02HYPM/R00005/instruments/GWFP_sn12997-01/0*/E*.DA?,GP02HYPM-WFP02-03-DOSTAL000,recovered_wfp,Available,
 mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00005/instruments/GWFP_sn12997-01/0*/C*.DA?,GP02HYPM-WFP02-04-CTDPFL000,recovered_wfp,Available,
-mi.dataset.driver.vel3d_l.wfp.vel3d_l_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00005/instruments/GWFP_sn12997-01/0*/A*.DA?,GP02HYPM-WFP02-05-VEL3DL000,recovered_wfp,Missing,Redmine 13629
+#mi.dataset.driver.vel3d_l.wfp.vel3d_l_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00005/instruments/GWFP_sn12997-01/0*/A*.DA?,GP02HYPM-WFP02-05-VEL3DL000,recovered_wfp,Not Expected,instrument failed on deployment
```
Comment (snwhite) [inline GP02HYPM/GP02HYPM_R00005_ingest.csv:11]: Looks good.

File: GP02HYPM/GP02HYPM_R00006_ingest.csv [hunk 5]
```diff
@@ -4,12 +4,12 @@ parser,filename_mask,reference_designator,data_source,status,notes
 mi.dataset.driver.sio_eng.sio.sio_eng_sio_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/cg_data/MSIOC_sn*/STATUS/STA*.DA?,GP02HYPM-RIM01-00-SIOENG000,recovered_host,Available,
 mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_sio_ct_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/cg_data/MSIOC_sn*/CTD/*.DAT,GP02HYPM-RIM01-02-CTDMOG039,recovered_host,Available,
 mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_ct_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/CTDMO_sn*/SBE37*.hex,GP02HYPM-RIM01-02-CTDMOG039,recovered_inst,Available,
-mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/GWFP_sn13104-02/0*/E*.DA?,GP02HYPM-WFP02-00-WFPENG000,recovered_wfp,Available,
+#mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/GWFP_sn13104-02/0*/E*.DA?,GP02HYPM-WFP02-00-WFPENG000,recovered_wfp,Pending,Can't parse data. Redmine 12792
```
Comment (snwhite) [inline GP02HYPM/GP02HYPM_R00006_ingest.csv:7]: Looks good.

File: GP02HYPM/GP02HYPM_R00006_ingest.csv [hunk 6]
```diff
@@ -4,12 +4,12 @@ parser,filename_mask,reference_designator,data_source,status,notes
...
 mi.dataset.driver.flord_l_wfp.flord_l_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/GWFP_sn13104-02/0*/E*.DA?,GP02HYPM-WFP02-01-FLORDL000,recovered_wfp,Available,
 mi.dataset.driver.dosta_ln.wfp.dosta_ln_wfp_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/GWFP_sn13104-02/0*/E*.DA?,GP02HYPM-WFP02-03-DOSTAL000,recovered_wfp,Available,
 mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/GWFP_sn13104-02/0*/C*.DA?,GP02HYPM-WFP02-04-CTDPFL000,recovered_wfp,Available,
 mi.dataset.driver.vel3d_l.wfp.vel3d_l_wfp_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/GWFP_sn13104-02/0*/A*.DA?,GP02HYPM-WFP02-05-VEL3DL000,recovered_wfp,Available,
-mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/GWFP_sn12936-02/0*/E*.DA?,GP02HYPM-WFP03-00-WFPENG000,recovered_wfp,Available,
+#mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/GWFP_sn12936-02/0*/E*.DA?,GP02HYPM-WFP03-00-WFPENG000,recovered_wfp,Pending,Can't parse data. Redmine 12792
```
Comment (snwhite) [inline GP02HYPM/GP02HYPM_R00006_ingest.csv:12]: Looks good.

File: GS02HYPM/GS02HYPM_R00003_ingest.csv [hunk 7]
```diff
@@ -9,8 +9,8 @@ mi.dataset.driver.flord_l_wfp.flord_l_wfp_recovered_driver,/omc_data/whoi/OMC/GS
 mi.dataset.driver.dosta_ln.wfp.dosta_ln_wfp_driver,/omc_data/whoi/OMC/GS02HYPM/R00003/instruments/GWFP_sn13104-07/00000/E*.DA?,GS02HYPM-WFP02-03-DOSTAL000,recovered_wfp,Available,
 mi.dataset.driver.ctdpf_ckl.wfp.ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/GS02HYPM/R00003/instruments/GWFP_sn13104-07/00000/C*.DA?,GS02HYPM-WFP02-04-CTDPFL000,recovered_wfp,Available,
 mi.dataset.driver.vel3d_l.wfp.vel3d_l_wfp_recovered_driver,/omc_data/whoi/OMC/GS02HYPM/R00003/instruments/GWFP_sn13104-07/00000/A*.DA?,GS02HYPM-WFP02-05-VEL3DL000,recovered_wfp,Available,
-mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GS02HYPM/R00003/cg_data/MSIOC_sn/WFP/E22*.DA?,GS02HYPM-WFP03-00-WFPENG000,recovered_wfp,Pending,parser needs to be updated - redmine 12792. lower profiler was lost - process data from the controller
+#mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/GS02HYPM/R00003/cg_data/MSIOC_sn/WFP/E22*.DA?,GS02HYPM-WFP03-00-WFPENG000,recovered_wfp,Pending,parser needs to be updated - redmine 12792. lower profiler was lost - process data from the controller
```
Comment (snwhite) [inline GS02HYPM/GS02HYPM_R00003_ingest.csv:12]: Looks good.

**Review by snwhite — CHANGES_REQUESTED** [review]
1) GA02HYPM D00002, the controller S/N should be 14880, with 00 and 01 after it. I know it doesn't matter since it is commented out, but it would be good to be consistent.
2) We will not be ingesting any telemetered ZPLSG data, so we may want to remove those lines from all of the the D0000x CSVs.

File: GA02HYPM/GA02HYPM_D00001_ingest.csv [hunk 8]
```diff
@@ -1,6 +1,4 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,,GA02HYPM-MPM01-02-ZPLSGA009,telemetered,Not Deployed,Not installed
-#,,GA02HYPM-MPM01-02-ZPLSGA010,telemetered,Not Deployed,Not installed
```
Comment (snwhite) [inline GA02HYPM/GA02HYPM_D00001_ingest.csv:3]: Looks good.

File: GA02HYPM/GA02HYPM_D00002_ingest.csv [hunk 9]
```diff
@@ -1,15 +1,13 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,,GA02HYPM-MPM01-02-ZPLSGA009,telemetered,Not Deployed,Not installed
-#,,GA02HYPM-MPM01-02-ZPLSGA010,telemetered,Not Deployed,Not installed
```
Comment (snwhite) [inline GA02HYPM/GA02HYPM_D00002_ingest.csv:3]: Looks good.

File: GA02HYPM/GA02HYPM_D00002_ingest.csv [hunk 10]
```diff
@@ -1,15 +1,13 @@
...
+#mi.dataset.driver.WFP_ENG.wfp_sio.wfp_eng_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GA02HYPM/D00002/*.we_wfp_1488001.dat,GA02HYPM-WFP03-00-WFPENG000,telemetered,Not Expected,Data not expected - glider not available to send data
+#mi.dataset.driver.flord_l_wfp.sio.flord_l_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GA02HYPM/D00002/*.we_wfp_1488001.dat,GA02HYPM-WFP03-01-FLORDL000,telemetered,Not Expected,Data not expected - glider not available to send data
+#mi.dataset.driver.dosta_ln.wfp_sio.dosta_ln_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GA02HYPM/D00002/*.we_wfp_1488001.dat,GA02HYPM-WFP03-03-DOSTAL000,telemetered,Not Expected,Data not expected - glider not available to send data
+#mi.dataset.driver.ctdpf_ckl.wfp_sio.ctdpf_ckl_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GA02HYPM/D00002/*.wc_wfp_1488001.dat,GA02HYPM-WFP03-04-CTDPFL000,telemetered,Not Expected,Data not expected - glider not available to send data
+#mi.dataset.driver.vel3d_l.wfp.sio.vel3d_l_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GA02HYPM/D00002/*.wa_wfp_1488001.dat,GA02HYPM-WFP03-05-VEL3DL000,telemetered,Not Expected,Data not expected - glider not available to send data
```
Comment (snwhite) [inline GA02HYPM/GA02HYPM_D00002_ingest.csv:13]: Looks good.

File: GA02HYPM/GA02HYPM_D00003_ingest.csv [hunk 11]
```diff
@@ -1,6 +1,4 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,,GA02HYPM-MPM01-02-ZPLSGA009,telemetered,Pending,Issues with data - MIO investigating
-#,,GA02HYPM-MPM01-02-ZPLSGA010,telemetered,Pending,Issues with data - MIO investigating
```
Comment (snwhite) [inline GA02HYPM/GA02HYPM_D00003_ingest.csv:3]: Looks good.

File: GI02HYPM/GI02HYPM_D00001_ingest.csv [hunk 12]
```diff
@@ -1,6 +1,4 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,,GI02HYPM-MPM01-02-ZPLSGA009,telemetered,Not Deployed,Not deployed
-#,,GI02HYPM-MPM01-02-ZPLSGA010,telemetered,Not Deployed,Not deployed
```
Comment (snwhite) [inline GI02HYPM/GI02HYPM_D00001_ingest.csv:3]: Looks good.

File: GI02HYPM/GI02HYPM_D00002_ingest.csv [hunk 13]
```diff
@@ -1,6 +1,4 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,,GI02HYPM-MPM01-02-ZPLSGA009,telemetered,Not Deployed,Not iinstalled
-#,,GI02HYPM-MPM01-02-ZPLSGA010,telemetered,Not Deployed,Not Installed
```
Comment (snwhite) [inline GI02HYPM/GI02HYPM_D00002_ingest.csv:3]: Looks good.

File: GI02HYPM/GI02HYPM_D00003_ingest.csv [hunk 14]
```diff
@@ -1,10 +1,8 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,/omc_data/whoi/OMC/GI02HYPM/D00003/node23p1.dat ,GI02HYPM-MPM01-02-ZPLSGA009,telemetered,Pending,data present in the master file but not ready for use - no parser available
-#,/omc_data/whoi/OMC/GI02HYPM/D00003/node23p1.dat ,GI02HYPM-MPM01-02-ZPLSGA010,telemetered,Pending,data present in the master file but not ready for use - no parser available
```
Comment (snwhite) [inline GI02HYPM/GI02HYPM_D00003_ingest.csv:3]: Looks good.

File: GI02HYPM/GI02HYPM_D00004_ingest.csv [hunk 15]
```diff
@@ -1,10 +1,8 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,/omc_data/whoi/OMC/GI02HYPM/D00004/node*p1_*.zp*09.dat,GI02HYPM-MPM01-02-ZPLSGA009,telemetered,Pending,no parser
-#,/omc_data/whoi/OMC/GI02HYPM/D00004/node*p1_*.zp*10.dat,GI02HYPM-MPM01-02-ZPLSGA010,telemetered,Pending,no parser
```
Comment (snwhite) [inline GI02HYPM/GI02HYPM_D00004_ingest.csv:3]: Looks good.

File: GI02HYPM/GI02HYPM_D00005_ingest.csv [hunk 16]
```diff
@@ -1,10 +1,8 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,/omc_data/whoi/OMC/GI02HYPM/D00005/node*p1_*.zp*09.dat,GI02HYPM-MPM01-02-ZPLSGA009,telemetered,Pending,no parser
-#,/omc_data/whoi/OMC/GI02HYPM/D00005/node*p1_*.zp*10.dat,GI02HYPM-MPM01-02-ZPLSGA010,telemetered,Pending,no parser
```
Comment (snwhite) [inline GI02HYPM/GI02HYPM_D00005_ingest.csv:3]: Looks good.

File: GI02HYPM/GI02HYPM_D00006_ingest.csv [hunk 17]
```diff
@@ -1,10 +1,8 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,/omc_data/whoi/OMC/GI02HYPM/D00006/node*p1_*.zp*09.dat,GI02HYPM-MPM01-02-ZPLSGA009,telemetered,Pending,no parser
-#,/omc_data/whoi/OMC/GI02HYPM/D00006/node*p1_*.zp*10.dat,GI02HYPM-MPM01-02-ZPLSGA010,telemetered,Pending,no parser
```
Comment (snwhite) [inline GI02HYPM/GI02HYPM_D00006_ingest.csv:3]: Looks good.

File: GI02HYPM/GI02HYPM_D00007_ingest.csv [hunk 18]
```diff
@@ -1,10 +1,8 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,/omc_data/whoi/OMC/GI02HYPM/D00007/node*p1_*.zp*09.dat,GI02HYPM-MPM01-02-ZPLSGA009,telemetered,Pending,no parser
-#,/omc_data/whoi/OMC/GI02HYPM/D00007/node*p1_*.zp*10.dat,GI02HYPM-MPM01-02-ZPLSGA010,telemetered,Pending,no parser
```
Comment (snwhite) [inline GI02HYPM/GI02HYPM_D00007_ingest.csv:3]: Looks good.

File: GP02HYPM/GP02HYPM_D00001_ingest.csv [hunk 19]
```diff
@@ -1,6 +1,4 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,,GP02HYPM-MPM01-02-ZPLSGA009,telemetered,Not Deployed,Not deployed
-#,,GP02HYPM-MPM01-02-ZPLSGA010,telemetered,Not Deployed,Not deployed
```
Comment (snwhite) [inline GP02HYPM/GP02HYPM_D00001_ingest.csv:3]: Looks good.

File: GP02HYPM/GP02HYPM_D00002_ingest.csv [hunk 20]
```diff
@@ -1,6 +1,4 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,,GP02HYPM-MPM01-02-ZPLSGA009,telemetered,Not Deployed,Not deployed
-#,,GP02HYPM-MPM01-02-ZPLSGA010,telemetered,Not Deployed,Not deployed
```
Comment (snwhite) [inline GP02HYPM/GP02HYPM_D00002_ingest.csv:3]: Looks good.

File: GP02HYPM/GP02HYPM_D00003_ingest.csv [hunk 21]
```diff
@@ -1,6 +1,4 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,,GP02HYPM-MPM01-02-ZPLSGA009,telemetered,Not Deployed,Not deployed
-#,,GP02HYPM-MPM01-02-ZPLSGA010,telemetered,Not Deployed,Not deployed
```
Comment (snwhite) [inline GP02HYPM/GP02HYPM_D00003_ingest.csv:3]: Looks good.

File: GP02HYPM/GP02HYPM_D00004_ingest.csv [hunk 22]
```diff
@@ -1,6 +1,4 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,,GP02HYPM-MPM01-02-ZPLSGA009,telemetered,Pending,Issues with data -  its in the mdd and nodeXXp1.dat files but MIOs think the data format was corrupted and the system won't be able to read it
-#,,GP02HYPM-MPM01-02-ZPLSGA010,telemetered,Pending,Issues with data -  its in the mdd and nodeXXp1.dat files but MIOs think the data format was corrupted and the system won't be able to read it
```
Comment (snwhite) [inline GP02HYPM/GP02HYPM_D00004_ingest.csv:3]: Looks good.

File: GP02HYPM/GP02HYPM_D00005_ingest.csv [hunk 23]
```diff
@@ -1,6 +1,4 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,,GP02HYPM-MPM01-02-ZPLSGA009,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
-#,,GP02HYPM-MPM01-02-ZPLSGA010,telemetered,Not Expected,No telemetered data expected (no gliders deployed)
```
Comment (snwhite) [inline GP02HYPM/GP02HYPM_D00005_ingest.csv:3]: Looks good.

File: GP02HYPM/GP02HYPM_D00006_ingest.csv [hunk 24]
```diff
@@ -1,6 +1,4 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,,GP02HYPM-MPM01-02-ZPLSGA009,telemetered,Pending,no parser
-#,,GP02HYPM-MPM01-02-ZPLSGA010,telemetered,Pending,no parser
```
Comment (snwhite) [inline GP02HYPM/GP02HYPM_D00006_ingest.csv:3]: Looks good.

File: GP02HYPM/GP02HYPM_D00007_ingest.csv [hunk 25]
```diff
@@ -1,15 +1,13 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,,GP02HYPM-MPM01-02-ZPLSGA009,telemetered,Pending,no parser
-#,,GP02HYPM-MPM01-02-ZPLSGA010,telemetered,Pending,no parser
```
Comment (snwhite) [inline GP02HYPM/GP02HYPM_D00007_ingest.csv:3]: Looks good.

File: GS02HYPM/GS02HYPM_D00001_ingest.csv [hunk 26]
```diff
@@ -1,6 +1,4 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,,GS02HYPM-MPM01-02-ZPLSGA009,telemetered,Not Deployed,Not deployed
-#,,GS02HYPM-MPM01-02-ZPLSGA010,telemetered,Not Deployed,Not deployed
```
Comment (snwhite) [inline GS02HYPM/GS02HYPM_D00001_ingest.csv:3]: Looks good.

File: GS02HYPM/GS02HYPM_D00002_ingest.csv [hunk 27]
```diff
@@ -1,6 +1,4 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,,GS02HYPM-MPM01-02-ZPLSGA009,telemetered,Not Deployed,Not deployed
-#,,GS02HYPM-MPM01-02-ZPLSGA010,telemetered,Not Deployed,Not deployed
```
Comment (snwhite) [inline GS02HYPM/GS02HYPM_D00002_ingest.csv:3]: Looks good.

File: GS02HYPM/GS02HYPM_D00003_ingest.csv [hunk 28]
```diff
@@ -1,6 +1,4 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,,GS02HYPM-MPM01-02-ZPLSGA009,telemetered,Not Expected,No glider deployed so telemetered data will not be available
-#,,GS02HYPM-MPM01-02-ZPLSGA010,telemetered,Not Expected,No glider deployed so telemetered data will not be available
```
Comment (snwhite) [inline GS02HYPM/GS02HYPM_D00003_ingest.csv:3]: Looks good.

**Review by snwhite — APPROVED** [review]
Additional changes look good!

**Review by kuo-james — APPROVED** [review]

**Comment by snwhite** [comment]
[Ingestion CSV Review [DT-129]](https://trello.com/c/RERKsbQ4/129-ingestion-csv-review-dt-129)

**Pull request merged**